api_end_end_testing/src/main/java/tds/user/api/model/RoleAssociation.java

```diff
@@ -0,0 +1,42 @@
+package tds.user.api.model;
```

Comment:
I don't know best practices for this myself so someone please jump in.  

Does it make sense for these to live in the test package instead since they are only used there?

@millerts @jeffjohnson9046 @burzum619